### DISCUSSION TITLE
Windows 7 icon for Windows 11

### DISCUSSION THREAD

**Discussion by TheJaredWilcurt** [Open-Shell/Open-Shell-Menu#1942] [Ideas]
My work laptop updated itself to Windows 11 today. The Win 7 icon I had been using for years doesn't quite work with Windows 11. You can see the Windows 11 logo *under* the Windows 7 logo poking out:

![Win 11 logo visible under Win 7 logo](https://github.com/Open-Shell/Open-Shell-Menu/assets/4629794/ec915f1c-0b2d-4990-8d19-7aefd213c973)

I measured it and it's off by 5 pixels, so I just added an extra 5 pixels to the left of the image, to move it over some and fully cover up the Windows 11 logo. Here's the new custom icon you can download:

![Windows 7 Start Button Small Icons](https://github.com/Open-Shell/Open-Shell-Menu/assets/4629794/f0aa6c56-399d-48d9-8adf-0f740a0d6054)

**NOTE:** This only works for when the task bar is left aligned. I tried to make a different icon for those that want it centered, but my offset trick wouldn't work. Good luck to anyone else that wants to try, I put about 20-30 minutes into it before giving up. But at least the left align version works.

**Comment by Mr-Sans-git** [comment]
I have gotten it to work with no align and a size of 60. Hope this helps to anyone else using this!
![image](https://github.com/user-attachments/assets/e431c410-8550-42b2-9534-24ae87b7edbf)

**Reply by Gurachn** [reply]
I've just installed on Win 11, and all seems good, but I want to use a custom start button.

It lets me select the stock Aero shell icon, but won't allow me to pick an image

**Reply by Gurachn** [reply]
> I've just installed on Win 11, and all seems good, but I want to use a custom start button.
> 
> It lets me select the stock Aero shell icon, but won't allow me to pick an image

So I can select either the Aero or Classic start buttons and they seem to work fine (if alignment slightly off)
But while I can select the custom  button, clicking on the 'Pick image...' has no effect.
<img width="1010" height="708" alt="image" src="https://github.com/user-attachments/assets/ab684a38-6ce1-4826-a737-e00a3a1dff82" />

Also, changing alignment or button size of the Aero or Classic icons also has no effect

**Comment by CTVCAM8** [comment]
Or use the next size-up button:

http://classicshell.net/forum/viewtopic.php?f=18&t=8565#p36406 


![3 Med size](https://github.com/user-attachments/assets/a97a972a-2f8e-49f6-a758-9e0612f10d74)

**Comment by ijustatemyTV** [comment]
bro are you guys start buttons on crack or smth its STILL UNALIGNED
<img width="314" height="195" alt="Screenshot 2025-12-13 123352" src="https://github.com/user-attachments/assets/97e5df8a-06c7-4d4e-ba37-e6abc3f6d499" />

**Comment by CTVCAM8** [comment]
Anything is possible with Microsoft changing code. 

Could be Microsoft has come out with a new version of their start menu\taskbar, 
which messes with Open-Shell Custom Button settings even more than before
or
 judging by the fuzzy edge of the globe image, 
it looks like the button size setting was increased, which also increased the left invisible padding part of the image.
If you don't want to use a bigger original button image, you could try to crop off the invisible pixels
 on the left side of your button image using a graphics editor.

**Reply by ijustatemyTV** [reply]
well i fixed it by editing the image to be more to the left or whatever direction is the opposite direction of the mis-alignment (im not good with directions :( )

**Reply by CTVCAM8** [reply]
Good for you [ijustatemyTV].

After recently playing around exchanging different size buttons,
 it seems like I have fallen into the rabbit hole of Win 11 updates.
I'm not able to change Custom Buttons anymore. 
for some help,
see https://github.com/Open-Shell/Open-Shell-Menu/issues/2364

**Comment by crxckd1** [comment]
How do yall do this i also want this
